index.ts

```diff
@@ -188,6 +198,39 @@ export function load(app: Application) {
 		const outFile =
 			app.options.getValue("coverageOutputPath") ||
 			join(event.outputDirectory, "coverage.svg");
-		writeFileSync(outFile, badge);
+		const outFileJson = outFile.replace(".svg", ".json");
```

Comment:
Should probably use a regex here to make sure we're replacing the extension, not some other svg like in `projects/renderer.svg/coverage.svg`

Reply:
Good point. Fixed.